lib/src/util/tested_expressions.dart

```diff
@@ -65,6 +65,7 @@ bool _sameOperands(String eLeftOperand, String bcLeftOperand,
   return sameOperandsSameOrder || sameOperandsInverted;
 }
 
+// ignore: camel_case_types
```

Comment:
Is there a reason to not just fix the name rather than ignoring the lint (here and elsewhere)?